README.md

```diff
@@ -97,147 +97,136 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+ - The PowerToys plugin allows calling the new Environment Variables utility. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Refactored and added support to VSCodium Stable, VSCodium Insider and Remote Tunnels workspaces. Thanks [@eternalphane](https://github.com/eternalphane)!
 
 ### Quick Accent
 
- - The "All languages" character set is now calculated by programmatically querying the characters for every available language. Thanks [@dannysummerlin](https://github.com/dannysummerlin)!
- - Added é to the Norwegian and Swedish languages. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
- - Added a runtime cache to the "All languages" character set, to only calculate accents once per key.
+ - Fixed characters that were removed from "All languages" because they were not in any single language. (This was a hotfix for 0.74)
+ - Added Asturian characters to the Spanish character set. Thanks [@blakestack](https://github.com/blakestack)!
+ - Added Greek characters with tonos. Thanks [@PesBandi](https://github.com/PesBandi)! 
 
 ### Registry Preview
 
- - Fixed focusing issues at startup.
- - Improved the data visualization to show data in a similar way to the Windows Registry Editor. Thanks [@dillydylann](https://github.com/dillydylann)!
+ - Fixed a parsing error that crashed the Application. (This was a hotfix for 0.74)
+ - Fixed opening file names with non-ASCII characters. Thanks [@randyrants](https://github.com/randyrants)!
+ - Fixed wrong parsing when the file contained an assignment with spaces around the equals sign. Thanks [@randyrants](https://github.com/randyrants)!
+ - Fixed key transversal issues when a key was a substring of a parent key. Thanks [@randyrants](https://github.com/randyrants)!
 
 ### Runner
 
- - Fixed hanging when a bug report was generated from the flyout. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Fixed the update notification toast to show an Unicode arrow. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!
 
 ### Settings
 
- - Improved the way the OOBE window reacts to Windows theme change.
- - Fixed an issue that made it impossible to change the "Switch between windows in the current zone" "Next window" shortcut for FancyZones.
- - Fixed a crash when entering a duplicate name for a color in the Color Picker page and improved clean up when cancelling a color edit. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Added a new Dashboard home page, with quick access for enabling modules, short descriptions and activation methods. Thanks [@niels9001](https://github.com/niels9001) for the design and UI work that made this possible!
+ - Fixed a typo in the Hosts File Editor page. Thanks [@Deepak-Sangle](https://github.com/Deepak-Sangle)!
+ - Added a lock icon to the flyout listing of all modules when its enabled state is controlled by policy.
 
-### Text Extractor
+### VCM
```

Comment:
Video Conference Mode

Reply:
Pushed a commit with "Video Conference Mute". Thank you!